### PR TITLE
Add google analytics events.

### PR DESCRIPTION
**Pull request by rjweeks**

### CODE REVIEW

**Comment by dcdarrell9** [comment]
if you have a look in eq, there might be a nicer way to add these in if we're going to add them in more places
https://github.com/ONSdigital/eq-survey-runner/search?utf8=%E2%9C%93&q=ga&type=

File: frontstage/templates/surveys/surveys-access.html [hunk 1]
```diff
@@ -30,7 +30,8 @@ <h2 class="neptune">Download</h2>
 
 
 <a id="download-survey-button" class="btn action-button btn--primary" target="_blank"
-   href="{{ url_for('surveys_bp.download_survey', cid=collection_instrument_id, case_id=case_id) }}">
+   href="{{ url_for('surveys_bp.download_survey', cid=collection_instrument_id, case_id=case_id) }}"
```
Comment (tej99) [inline frontstage/templates/surveys/surveys-access.html]: Doesn't look like cid=collection_instrument_id is actually needed. download_survey just requires case_id.

File: frontstage/views/surveys/upload_survey.py [hunk 2]
```diff
@@ -58,6 +58,7 @@ def upload_survey(session):
         return redirect(url_for('surveys_bp.upload_failed',
                                 _external=True,
                                 case_id=case_id,
+                                party_id=party_id,
```
Comment (tej99) [inline frontstage/views/surveys/upload_survey.py]: Might not need to pass the party_id

File: frontstage/views/surveys/upload_survey_failed.py [hunk 3]
```diff
@@ -14,7 +15,17 @@
 @jwt_authorization(request)
 def upload_failed(session):
     case_id = request.args.get('case_id', None)
+    party_id = request.args.get('party_id', None)
```
Comment (tej99) [inline frontstage/views/surveys/upload_survey_failed.py]: Can get party_id from the session instead of passing it in. party_id = session['party_id']

**Pull request merged**